enclave/core/CMakeLists.txt

```diff
@@ -66,6 +66,18 @@ endif()
 set(PLATFORM_SRC "")
 set(MUSL_SRC_DIR ${PROJECT_SOURCE_DIR}/3rdparty/musl/musl/src)
 
+if (USE_SNMALLOC)
+    add_library(oeallocator OBJECT
```

Comment:
Per recent changes, please use corresponding wrappers instead of built-in cmake functions. For example, using `add_enclave_library` instead of `add_library`. See https://github.com/openenclave/openenclave/blob/master/cmake/enclave_cmake_wrappers.cmake for more details.

Reply:
Thank you for pointing this out. Fixed.